admin/README.md

```diff
@@ -9,10 +9,18 @@ curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"
 ```
 
 ### To change log level
+Flow, and other zerolog-based libraries:
+
 ```
 curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"commandName": "set-log-level", "data": "debug"}'
 ```
 
+libp2p, badger, and other golog-based libraries:
+
+```
+curl localhost:9002/admin/run_command -H 'Content-Type: application/json' -d '{"commandName": "set-golog-level", "data": "debug"}'
```

Comment:
I have a really dumb question.  why are we doing this / why not just send a grpc request via a command line tool?  This seems like a poor man's rpc system ...

Reply:
@jordanschalm did we consider using grpc for the admin interface?